.muse/install_eslint_plugins.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+npm install @typescript-eslint/eslint-plugin@latest --save-dev
```

Comment:
Minor: we use `yarn` everywhere in this repo not `npm`

Reply:
I can try `yarn`, but Javascript/Typescript projects don't seem to be built by muse.dev so it may just be running eslint via npm irregardless of what the project uses.  https://docs.muse.dev/docs/repository-configuration/ lists the project types that use a build system.